lib/integrationTools.js

```diff
@@ -47,18 +47,26 @@ const getPullRequest = ({pullUrl, fullName, branchName, installationId}) => {
 const getGiphyFromText = (text) => {
   return new Promise((resolve, reject) => {
     var searchTerms = encodeURIComponent(text)
-    var giphySearchUrl = 'http://api.giphy.com/v1/gifs/search?q=' + searchTerms + '&api_key=' + GIPHY_API_KEY
+    var giphySearchUrl = 'http://api.giphy.com/v1/gifs/search?rating=pg&q=' + searchTerms + '&api_key=' + GIPHY_API_KEY
     var giphyGifUrl = 'http://i.giphy.com/'
 
     request.get(
       {url: giphySearchUrl},
       (err, res, body) => {
         const {data} = JSON.parse(body)
         if (data && data.length) {
-          let randomOne = data[Math.floor(Math.random() * data.length)]
-          resolve('![](' + giphyGifUrl + randomOne.id + '.' + randomOne.type + ')')
+          const lessThan5Mb = data.filter((gif) => {
+            return gif.images &&
+                   gif.images.original &&
+                   parseInt(gif.images.original.size) < 5000000
+          })
+          let randomOne = lessThan5Mb[Math.floor(Math.random() * lessThan5Mb.length)]
+          let messageString =
+            ? '![](' + giphyGifUrl + randomOne.id + '.' + randomOne.type + ')'
+            : 'gif is too big for github :broken_heart:'
+          resolve(messageString)
         } else {
-          reject()
+          reject('something went wrong with giphy api')
```

Comment:
yeah probably should do that - good idea! - but I will make an issue for the error messages since we should be doing this: `reject(new Error('blah blah'))`